examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -930,6 +931,10 @@ def main(args):
             images = pipeline(prompt, num_inference_steps=25, generator=generator).images
 
             for tracker in accelerator.trackers:
+                if tracker.name == "tensorboard":
+                    np_images = np.stack([np.asarray(img) for img in images])
+                    np_images = np_images.transpose(0, 3, 1, 2)
```

Comment:
Because `SummaryWriter` expects `CHW` format by default, and `np.asarray` converts PIL images using channels-last.

We could do without the transpose and use the following instead:

```Python
tracker.writer.add_images("validation", np_images, epoch, dataformats='NHWC')
```

Do you think that'd be clearer?

Reply:
Thanks for explaining! Yes, I think that is clearer.